python/sqlflow_submitter/tensorflow/pai_distributed.py

```diff
@@ -51,9 +51,9 @@ def set_oss_environs(FLAGS):
     os.environ["SQLFLOW_OSS_MODEL_ENDPOINT"] = FLAGS.sqlflow_oss_ep
 
 
-# make_distributed_info_without_evaluator and dump_into_tf_config are used to dump
-# distributed configurations into environment variable TF_CONFIG so that Tensorflow
-# can recognize it.
+# make_distributed_info_without_evaluator and dump_into_tf_config are used to
+# dump distributed configurations into environment variable TF_CONFIG so that
+# Tensorflow can recognize it.
```

Comment:
Tensorflow => TensorFlow

Reply:
I believe this line was formatted at pre-commit. However, let me fix the typo.